bioc_intro.Rmd

```diff
@@ -0,0 +1,28 @@
+# Introduction to Bioconductor Classes {-}
+
+**S4 system**
+
+S4 class system has brought several useful features to the object-oriented programming paradigm within R.
+It has been constantly deployed in R packages, including the [Bioconductor](https://bioconductor.org/) package.
+
+|   Books:
+
+* John M. Chambers. Software for Data Analysis: Programming with R. Springer, New York, 2008. ISBN-13 978-0387759357.
+* I Robert Gentleman. R Programming for Bioinformatics. Chapman & Hall/CRC, New York, 2008. ISBN-13 978-1420063677
+    
+|   Online Document:
+
+* Hervé Pagès, “A quick overview of the S4 class system”,\
+ https://bioconductor.org/packages/release/bioc/vignettes/S4Vectors/inst/doc/S4QuickOverview.pdf
+* Laurent Gatto, “A practical tutorial on S4 programming”,\
+ https://bioconductor.org/help/course-materials/2013/CSAMA2013/friday/afternoon/S4-tutorial.pdf
+* John M. Chambers. “How S4 Methods Work”,\
+ http://developer.r-project.org/howMethodsWork.pdf
```

Comment:
I would just add links [like this](url). No need to show the exact URLs as text.